openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Doris.md

```diff
@@ -0,0 +1,76 @@
+# Doris
+
+In this section, we provide guides and references to use the Doris connector.
+
+## Requirements
+
+You can find further information on the Doris connector in the [docs](https://docs.open-metadata.org/connectors/database/doris).
```

Comment:
Thanks for the pr @chyueyi, this link would currently not work as we have not added the docs in the docs dir, do you plan add the same with this pr? 🙇

Reply:
OK @ulixius9